index.html

```diff
@@ -130,6 +159,9 @@ <h3 class="project-heading">Multi-Post Stories Gain+Glory</h3>
 
       <section class="section-about">
         <section class="about-header">
+          <div class="illus1">
+            <img src="./images/illustration-about-me-1-desktop.png" alt="" />
+          </div>
```

Comment:
- [ ] Please make sure all these `divs` with `class='illus'` are used as background images as these images are not part of the main content they should be applied as a `background-image` and not with `img` tags, please do this for all your `divs` which is used for the image, Just like you have done for the headline section